test/boost/cql_query_test.cc

```diff
@@ -5659,8 +5659,9 @@ SEASTAR_TEST_CASE(test_null_and_unset_in_collections) {
 
         auto where_in_list_marker = e.prepare("SELECT * FROM null_in_col WHERE p IN ?").get0();
 
-        BOOST_REQUIRE_EXCEPTION(e.execute_prepared(where_in_list_marker, {list_with_null}).get(),
-                                exceptions::invalid_request_exception, check_null_msg());
+        auto msg = e.execute_prepared(where_in_list_marker, {list_with_null}).get();
+        assert_that(msg).is_rows().with_rows({});
```

Comment:
Maybe the commit message should mention this change - that now IN allows NULL (which I guess this NULL is ignored, right?)

Reply:
Yes. More properly, the NULL is compared against the column value. Since NULL does not equal NULL, there can be no matches, so the result is as if it was ignored.